pom.xml

```diff
@@ -128,6 +128,7 @@
         <testcontainer.version>1.17.6</testcontainer.version>
         <spotless.version>2.29.0</spotless.version>
         <jsqlparser.version>4.5</jsqlparser.version>
+        <jsonpath.version>2.8.0</jsonpath.version>
```

Comment:
There already exist `2.7.0`, it's not a good idea to used multiple version `json-path`.

Reply:
not found a dependency for 2.7.0 yet.Can you point out where it is thank you

Reply:
https://github.com/apache/seatunnel/blob/a15b79f5ac8b49e969a64601d9b9436003d0d0d3/seatunnel-connectors-v2/connector-http/connector-http-base/pom.xml#L32-L37